src/main/java/nl/ordina/jobcrawler/service/ScraperService.java

```diff
@@ -12,6 +12,7 @@
 import org.springframework.scheduling.annotation.Scheduled;
 import org.springframework.stereotype.Service;
 
+import javax.annotation.PostConstruct;
```

Comment:
```suggestion
```
as the @PostConstruct is commented. Maybe the annotation should also be removed

Reply:
adapted